### DISCUSSION TITLE
Feature Request: Browser Timezone option for Elasticsearch date histogram

### DISCUSSION THREAD

**Discussion by krische** [grafana/grafana#63318] [Data Sources]
# What would you like to be added

Somewhat recently, a feature was added to specify the time zone used for Elasticsearch Date Histogram with #40882. However, it looks like there is no option for a "Browser's Time Zone"; there is only hard-coded options like UTC, etc. This can cause confusion for users, especially when dealing with larger/wider intervals like `1d`, because it can make data points to appear incorrect by appearing on different days.

I would like there to be an option that has it use the user's browser's time zone.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82852. Feel free to continue the discussion around this there. Thank you!
